diagnostic.md

```diff
@@ -5,74 +5,81 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+To be able to store data and persist it over time.
```

Comment:
Good, also allows multiple users to interact over the same app on different devices.